radiography/src/main/java/radiography/compose/ComposeLayoutRenderers.kt

```diff
@@ -0,0 +1,132 @@
+package radiography.compose
+
+import androidx.compose.ui.layout.LayoutIdParentData
+import androidx.compose.ui.platform.InspectableParameter
+import androidx.compose.ui.semantics.SemanticsModifier
+import androidx.compose.ui.semantics.SemanticsProperties
+import androidx.compose.ui.semantics.SemanticsProperties.Text
+import androidx.compose.ui.semantics.SemanticsPropertyKey
+import radiography.AttributeAppendable
+import radiography.TypedViewStateRenderer
+import radiography.ViewStateRenderer
+import radiography.ellipsize
+
+// Note that this class can't use viewStateRendererFor, since that function is defined in the
+// ViewStateRenderers object, whose class initializer may initialize _this_ class, which will cause
+// NoClassDefFoundExceptions. This can happen when a debugger is attached.
```

Comment:
what fresh hell is this? Are you saying that if ComposeLayoutRenderers uses a function defined in ViewStateRenderers, then when loading ViewStateRenderers the classloader may also also load ComposeLayoutRenderers? If yes, something's wrong with the compiler..

Reply:
btw, why would initializing this class cause NoClassDefFoundExceptions ? Looks like everything here has `if (!isComposeAvailable) NoRenderer`